packages/table/src/Controls/TableControls.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { cloneElement, isValidElement, useEffect, useState } from 'react';
+import { Form } from 'reactstrap';
+
+type Props = {
+  id?: string;
+  disabled?: boolean;
+  children?: React.ReactNode[] | React.ReactNode;
+} & React.HTMLAttributes<HTMLElement>;
+
+const TableControls = ({ id, disabled, children, ...rest }: Props): JSX.Element => {
+  const [isDisabled, setIsDisabled] = useState<boolean>(disabled || false);
```

Comment:
It's ok to leave out the `<boolean>` here. It will get inferred from `disabled`.